Example/Backpack/SwiftUI/Components/Rating/RatingExampleViewWithImage.swift

```diff
@@ -0,0 +1,64 @@
+//
```

Comment:
Here as well, please remove the //

Reply:
updated in  27fa2c175db76551ee1eeae2b1f720dd65151395